lib/challenge.js

```diff
@@ -52,6 +69,19 @@ var some_pet;
 
 // Answer Starts Here
 
+var some_pets = () {
+  name: 'DiDi',
+  species: 'dog',
+  breed: 'labrador/beagle mix',
+  noise: [
+    {
+      happyNoise: 'bark',
+      angryNoise: 'growl',
+    }
+  ],
```

Comment:
What you've assigned to the `noise` attribute here is a list of one element. It doesn't look like there's any purpose for additional elements, so it raises the question of why it's a list.

The question was looking for you to place a method on the object by assigning a function to the property `makeNoise`. Since what you've got between the curly braces is mostly good, I'll give this partial credit.